ci/jjb/jobs/projects.yaml

```diff
@@ -124,6 +128,7 @@
     upgrade_pulp_version:
         - 2.15-beta
```

Comment:
I think this line can be dropped. We (nominally) test whether it's possible to upgrade from the prior stable release to the current stable release.